modules/mri_violations/php/NDB_Menu_Filter_Form_mri_violations.class.inc

```diff
@@ -147,6 +147,13 @@ class NDB_Menu_Filter_Form_mri_violations extends NDB_Menu_Filter_Form
         else{
             $useProjects = true;
         }
+        // Get sites
+        $sitesCond = '';
+        if (!$user->hasPermission('access_all_profiles') 
+            && !is_null($user->getData('CenterID'))
+        ) {
+            $sitesCond = " AND p.CenterID = '".$user->getData('CenterID')."' ";
```

Comment:
This needs to be quoted with DB->quote, otherwise there's a potential SQL injection attack.